docs/build-n-test.md

```diff
@@ -22,6 +22,7 @@
     - ``BUILD_COMPAT``: build extra cCurl compatible interface.
     - ``BUILD_STAT``: show the statistics of the PoW information.
     - ``BUILD_DEBUG``: dump verbose messages internally.
+    - ``BUILD_SAN``: build with Sanitizer to detect software potential bugs. Available values are `address`, `undefined` and `thread`.
```

Comment:
How about migrating all relevant work for `BUILD_SAN` into `BUILD_DEBUG`? We can have both release and debug build.

Reply:
Like remove the `BUILD_SAN` option and use `BUILD_DEBUG` option instead?
For example: The `BUILD_DEBUG` available option would be
- 0
- 1
- address
- undefined
- thread

Reply:
I agree with the value range. The build with sanitizer(s) implies the set `CFLAGS += -Og -g3 -DENABLE_DEBUG`.

Reply:
Here is what I'm going to do.
The `BUILD_SAN` option would be no longer existed.
Instead, the `BUILD_DEBUG` option will be used and the available option would extend to the corresponding Sanitizer.

The document will be modified as well.

Reply:
Done.